microsoft-outlook/latest/support/index.md

```diff
@@ -2,10 +2,8 @@
 title: Supported platforms
 ---
 
-The following are the supported platforms for the Outlook Integration version 2.7:
+The following are the supported platforms for Outlook Integration 2.8:
 
 | Version | Notes |
 | ------- | ----- |
-| Alfresco Content Services 6.2.x | |
-| Alfresco Content Services 6.1.x | |
-| Alfresco Content Services 6.0.x | |
+| Alfresco Content Services 7.0.x | |
```

Comment:
6.2 is also supported with but needs some work. The T-Engine needs to be installed and these properties need to be set: -Dlegacy.transform.service.enabled=false
-Dlocal.transform.service.enabled=true
Otherwise the transformation of msg and eml files to pdf won't work, but it will install.

Anything before 6.2 will need an additional module which needs to be developed. This will make it possible to have transformations with plugin 2.8 and ACS below 6.2.

Reply:
Thanks for the review @mhofmann-itnovum. I've corrected the supported version in PR #221, and will add the extra config required for 6.2.2 in PR #216.